spec/factories/rubric_criteria.rb

```diff
@@ -5,16 +5,12 @@
     max_mark { 4.0 }
     ta_visible { true }
     peer_visible { false }
-    level_0_name { 'Poor' }
-    level_0_description { 'This criterion was not satisifed whatsoever.' }
-    level_1_name { 'Satisfactory' }
-    level_1_description { 'This criterion was satisfied.' }
-    level_2_name { 'Good' }
-    level_2_description { 'This criterion was satisfied well.' }
-    level_3_name { 'Great' }
-    level_3_description { 'This criterion was satisfied very well.' }
-    level_4_name { 'Excellent' }
-    level_4_description { 'This criterion was satisfied exceptionally well.' }
     sequence(:position)
   end
+
+  factory :rubric_criteria_with_levels do
+    after(:create) do |criteria|
+      5.times.each { |i| create(:level, rubric_criteria: rubric_criteria, mark: i) }
```

Comment:
Two issues here. First, rename the **key** from `rubric_criteria:` to `rubric_criterion:`. Second, right now you're passing in `rubric_criteria`, but what you really want is the current criterion value, which you've named `criteria` on Line 12 (i.e., the current object is passed to the block in `after(:create)`. But also you should rename the variable to `criterion` rather than `criteria`.